pkg/operator/osd/log.go

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright 2016 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package osd
+
+import (
+	"github.com/coreos/pkg/capnslog"
+)
+
+var logger = capnslog.NewPackageLogger("github.com/rook/rook-operator", "osd")
```

Comment:
the pod logs are all separate, but that's a good idea to maintain unique names in case the logs are aggregated